source-git/shutdown.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Service integration for CentOS Stream shut down
+date: 2024-08-30T08:00:00+02:00
+sidebar_position: 3
+---
+# Service integration for CentOS Stream shut down
+
+The Packit team wants to let all its users know that the cluster, where the *source-git service* was running, has sadly been shut down.
```

Comment:
I am wondering whether we have some docs or something that we could link here, for the *source-git service* part

Reply:
I can't find anything.

Reply:
we could probably link [this](https://packit.dev/source-git/status#service-integration-for-centos-stream)

Reply:
I know, I just meant the specific section

Reply:
yes, in here add the suffix, for the last sentence have it generally